src/migration/1723670727196-UserStreakAction.ts

```diff
@@ -5,7 +5,7 @@ export class UserStreakAction1723670727196 implements MigrationInterface {
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `CREATE TABLE "user_streak_action" ("userId" character varying(36) NOT NULL, "type" text NOT NULL, "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), CONSTRAINT "PK_875298ad33fc69f7d704042cdd4" PRIMARY KEY ("userId", "type", "createdAt"))`,
```

Comment:
Are we gonna manually update this in the DB? ..since this change in migration won't be run on deployment

Reply:
Yeah, this is wrong. I initially thought the migration had not yet run in prod. Let me create a new migration.